metron-platform/metron-common/src/main/java/org/apache/metron/common/writer/BulkWriterMessage.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.metron.common.writer;
+
+import java.util.Objects;
+
+public class BulkWriterMessage<MESSAGE_T> {
```

Comment:
Can you add javadoc to this class?

Reply:
Maybe it is just me and my dyslexic tendencies, but having both a `BulkWriterMessage` and `BulkMessageWriter` is really confusing me as I navigate through the code.  

Would naming this class a `BulkMessage` be more clear?  It makes sense to me that a `BulkMessageWriter` is responsible for writing `BulkMessages`.

I am totally willing to accept that this is just some weird short circuit in my brain and the name you have works just fine. :)

Reply:
I agree with you, it's confusing.  I like your idea better.

Reply:
@nickwallen I had the same double-take when I first read that. Good suggestion.